package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "alinka-website",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/CodeForPoznan/alinka-website.git"
+  },
+  "author": "",
```

Comment:
We could set it to CodeForPoznan like here: https://github.com/CodeForPoznan/alinka-electron/pull/100/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R5